design/envoy-json-logging-custom-fields-design.md

```diff
@@ -0,0 +1,144 @@
+# Allow Envoy to output arbitrary fields for JSON logs
+
+Status: Draft
```

Comment:
If there are no further comments, @mike1808, could you change this to 'Accepted' before we merge?

Reply:
Done @youngnick